src/commands/general/lookup.ts

```diff
@@ -20,7 +20,7 @@ export default class LookupCommand extends Command {
 		const ip = args[0];
 		if (!ip || !IP_REGEX.test(ip)) throw new CommandError('PROVIDE_IP');
      
-		const response = await fetch(URLs.IP_API(ip, 34809));
+		const response = await fetch(URLs.IP_API(ip, 59387));
```

Comment:
has the API changed ? why has this number been changed

Reply:
The bitfield was wrong you had, so i corrected it so i can resolve the field 'query'